sdk/src/core/mod.rs

```diff
@@ -51,9 +51,9 @@ pub struct Sdk {
     /// Contains the user repository for storing and loading different users.
     repo: Option<UserRepoT>,
     /// The currently active network
-    network: Option<Network>,
+    network: Option<ApiNetwork>,
```

Comment:
could we rename to `active_network` for better clarity? 😄  

```suggestion
    active_network: Option<ApiNetwork>,
```